### DISCUSSION TITLE
Make Order of Tags on Top Tags Page (/tags) Easier to Understand

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#16263] [Feature Requests]
It's currently tough to understand how the tags are ordered on the Top Tags page (e.g. dev.to/tags). They're using the "hotness score", but there's some weird behavior going on that occasionally makes them seem out of order.

It would be an improvement if these tags were simply ordered in a straightforward way by default (like number of posts published under the tag)... currently on the tags page it does list # of posts published for each individual tag (see image below) so it seems to suggest that it should be order from greatest to least. 

![Screen Shot 2022-01-21 at 3 11 18 PM](https://user-images.githubusercontent.com/16007075/150593772-10fff0ab-6848-431d-a1f5-1f74b246a812.png)

It would be an even bigger/better improvement if the users could sort and filter the page based on a few factors like: alphabetical order,  number of posts published, & number of followers (we could potentially think of more). Maybe when the user sorts to number of followers, this number could even be displayed on each tag similar to how number of posts published currently is displayed. Maybe if they sorted alphabetical the tag would simply display the tag's short summary.

**Comment by jeremyf** [comment]
I definitely like moving sorting away from hotness_score and towards "one with most articles".  Also I think sorting by various mechanisms would be great.

These all seem like good things to do.
